CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ### Enhancements
+* [vra] Issue 250 / property-group is imported with scope of shared project but no way of a global scope.
```

Comment:
@Tchoui thanks for taking the time to tackle this issue and also updating the structure and styling of the class to make it compatible with the others. 

Can I just ask for 2 updates on the documentation side:
1. To update the CHANGELOG entry to something like [vra] 250 / Add support for global scope property-group.
2. Add a short entry to the release.md in the docs/versions/latest/Release.md, e.g.:
https://github.com/vmware/build-tools-for-vmware-aria/pull/281/files#diff-2758ec22240769ca24b53db71e3dc4dc4a7990dabed3bc38bf2ec4456d19f3a1
or 
https://github.com/vmware/build-tools-for-vmware-aria/pull/277/files

The Release.md itself has fields suggesting what should be described based on the issue type